### DISCUSSION TITLE
Tibber Pulse

### DISCUSSION THREAD

**Discussion by duffel90** [tbnobody/OpenDTU#1322] [Q&A]
Hi, ist es möglich den Tibber Pulse über die API als Nulleinspeisung zu nutzen?
Beste Grüße

**Comment by NF1971** [comment]
Das geht, ist aber umständlich. Auf meiner Synology läuft HA (Grafana, InfluxDB, Node-Red). Die Nulleinspeisung wird über Node-Red umgesetzt. Der Umweg über den Tibber-Server kostet Zeit und führt dazu, dass die "Null" nicht ganz steht. Optimal wäre es direkt in OpenDTU den Tibber Lesekopf per http request lokal auszulesen.  Aber dafür gibt es wohl zuviele verschiedene Systeme (Tibber, Poweropti oder einfach einen EM24).
